### PR TITLE
feature/eugene/dat-2776/only-convert-to-datetime-if-valid

### PR DESCRIPTION
**Pull request by eugsokolov**

### CODE REVIEW

**Comment by eugsokolov** [comment]
maybe we should add a test...

**Comment by eugsokolov** [comment]
the issue is that the "value" field for a certain df (57588c68-e262-49b4-b05a-8ae4c30c183b) which is of type date is either a datetime or the string None

File: datamonster_api/lib/datamonster.py [hunk 1]
```diff
@@ -276,7 +276,9 @@ def _avro_to_df(self, avro_buffer, data_types):
 
         def parse_row(row):
             return {
-                col["name"]: pandas.to_datetime(row[col["name"]])
+                col["name"]: pandas.to_datetime(
+                    row[col["name"]] if row[col["name"]] != "None" else None
```
Comment (wbrinkerhoff) [inline datamonster_api/lib/datamonster.py]: Checking for a hardcoded string seems sketchy here, maybe it makes more sense to try to parse it and then do None if the parsing fails?  Also, I'm not sure we should support misconfigured data.  `None` is clearly not a date.

Comment (eugsokolov) [reply datamonster_api/lib/datamonster.py]: updated

**Pull request merged**